src/main/java/pl/arturzgodka/jsonmappers/SkillMapper.java

```diff
@@ -0,0 +1,70 @@
+package pl.arturzgodka.jsonmappers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import pl.arturzgodka.apihandlers.BaseUrlParts;
+import pl.arturzgodka.apihandlers.SkillHandlerApi;
+import pl.arturzgodka.datamodel.HeroSkillDataModel;
+import pl.arturzgodka.datamodel.SkillDataModel;
+import pl.arturzgodka.token.FetchToken;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkillMapper {
+
+    private HeroSkillDataModel fetchSkill(JsonNode node) {
```

Comment:
does this fetch anything or is it only mapping?

Reply:
only mapping, namming changed.